improver_tests/acceptance/acceptance.py

```diff
@@ -134,7 +134,7 @@ def recreate_if_needed(output_path, kgo_path, recreate_dir_path=None):
         print("Original KGO file does not exist")
     kgo_relative = kgo_path.relative_to(kgo_root_dir)
     recreate_file_path = recreate_dir_path / kgo_relative
-    if recreate_file_path.resolve() == kgo_path.resolve():
+    if recreate_file_path == kgo_path:
```

Comment:
I'm a bit confused about why you need to compare without resolving symlinks - I would have thought we'd need to make sure the absolute paths are different.

Reply:
What happens here is that if `recreate_file_path` contains a symlink to the ppdev repository `kgo_path`, `resolve()` says they are both the same location and prevents recreation of KGO.  Since this is the main use of RECREATE_KGO, it means you can't use this script without deleting all the necessary symlinks one by one.

A few lines down from this, `recreate_file_path` is unlinked.  This means it is safe to recreate KGO at this path even if it contains a symlink to the `kgo_path`, as the new KGO is not recreated at the source of the symlink.